delfin/drivers/hitachi/hnas/nas_handler.py

```diff
@@ -0,0 +1,825 @@
+                            eth_model['health_status'] = \
+                                constants.PortHealthStatus.UNKNOWN
+                            if value_info[0].split(":")[1] == 'UP':
+                                eth_model['connection_status'] = \
+                                    constants.PortConnectionStatus.CONNECTED
+                                eth_model['health_status'] = \
+                                    constants.PortHealthStatus.NORMAL
+                            eth_model['type'] = constants.PortType.ETH
+                            eth_model['storage_id'] = storage_id
+                            eth_model['native_port_id'] =\
+                                'ETH' + '-' + eth_model['name']
+                    if 'inet addr' in value:
+                        value_info = value.split()
+                        if len(value_info) > 2:
+                            eth_model['ipv4'] = value_info[1].split(":")[1]
+                            eth_model['ipv4_mask'] = \
+                                value_info[3].split(":")[1]
+                else:
+                    if 'name' in eth_model:
+                        eth_list.append(eth_model)
+                        eth_model = {}
+            return eth_list
+        except exception.DelfinException as e:
+            err_msg = "Failed to get eth ports from " \
+                      "hitachi nas: %s" % (six.text_type(e.msg))
+            LOG.error(err_msg)
+            raise e
+        except Exception as err:
+            err_msg = "Failed to get eth ports from " \
+                      "hitachi nas: %s" % (six.text_type(err))
+            LOG.error(err_msg)
+            raise exception.InvalidResults(err_msg)
+
+    def list_filesystems(self, storage_id):
+        try:
+            fs_list = []
+            fs_info = self.ssh_do_exec([constant.FS_INFO_COMMAND])
+            fs_array = self.get_table_data(fs_info)
+            status_info = self.ssh_do_exec([constant.FS_STATUS_COMMAND])
+            status_array = self.get_table_data(status_info)
+            status_map = {}
+            for status in status_array:
+                status_info = status.split()
+                if len(status_info) > 6:
+                    status_map[status_info[1]] = \
+                        [status_info[2], status_info[3]]
+            for fs in fs_array:
+                fs_info = list(filter(None, fs.split("  ")))
+                if len(fs_info) > 8:
+                    total_capacity = fs_info[3].replace(" ", '')
+                    used_capacity = fs_info[4].replace(" ", '').split("(")[0]
+                    free_capacity = fs_info[7].replace(" ", '').split("(")[0]
+                    total_capacity = self.get_size(total_capacity)
+                    used_capacity = self.get_size(used_capacity)
+                    free_capacity = self.get_size(free_capacity)
+                    type = constants.VolumeType.THICK \
```

Comment:
`type` is a built-in name, please use other variable name.

Reply:
has been  modify